src/data/selector.ts

```diff
@@ -124,6 +125,14 @@ export interface ConstantSelector {
   } | null;
 }
 
+export interface CountrySelector {
+  country: {
```

Comment:
I think this should have a `countries?: string[];` option, like the language selector has?